src/backend/gdb_expansion.ts

```diff
@@ -231,6 +243,15 @@ export function expandValue(variableCreate: (arg: VariableObject | string, optio
 			ref = variableCreate(getNamespace(name));
 			val = "...";
 		}
+		value = value.trim();
+		if (value[0] == ','){
+			let tmp = value;
+			tmp =  tmp.substring(1).trim();
+			if(tmp.startsWith("<No data fields>")){
+				value = tmp.replace(/^<No data fields>/, "");
+			}
+		}
```

Comment:
Isn't that just the following?

```ts
value = value.replace(/^, <No data fields>/, "");
```

The one question open - I haven't recognized that before: `value` is an outer variable, maybe it should be passed in via the lambda and also be returned?

But I'm also not sure: How is `value:val` in the return related to changing `value` directly?

Reply:
Isn't that just the following?

value = value.replace(/^, <No data fields>/, "");
that's cool. i tested, work fine.

The value of val has already been correctly obtained before the processing of value = value.replace(/^, <No data fields>/, "");. The processing with value = value.replace(/^, <No data fields>/, ""); is necessary because not removing ", <No data fields>" will affect the subsequent parsing.